osu.Game.Modes.Osu/Objects/Drawables/DrawableHitCircle.cs

```diff
@@ -69,15 +69,18 @@ public DrawableHitCircle(OsuHitObject h) : base(h)
             Size = circle.DrawSize;
         }
 
-        double hit50 = 150;
-        double hit100 = 80;
-        double hit300 = 30;
+        //todo: these aren't constants.

```

Comment:
Why did you replace them with constants then? hit50, hit100 and hit300 should remain per-hitobject members imho.